pages/resources/user.mdx

```diff
@@ -83,6 +84,7 @@ For users with migrated accounts, default avatar URLs will be based on the user
 | verified ^2^             | boolean                                                             | Whether the email on this account has been verified                                                             |
 | email ^2^                | ?string                                                             | The user's email address                                                                                        |
 | phone? ^1^               | ?string                                                             | The user's E.164-formatted phone number                                                                         |
+| premium ^4^              | boolean                                                             | Whether the user is subscribed to Nitro                                                                         |
```

Comment:
I didn't include this because it's deprecated but I guess there's no reason not to
```suggestion
| premium **(deprecated)** ^4^ | boolean                                                             | Whether the user is subscribed to Nitro                                                                         |
```